main.go

```diff
@@ -48,8 +48,15 @@ func rssHandler(sourceFeeds []sourceFeed) http.Handler {
 
 		sort.Sort(byCreated(master.Items))
 
-		result, _ := master.ToAtom()
-		fmt.Fprintln(rw, result)
+		result, err := master.ToAtom()
+		if err != nil {
+			log.Fatal(err)
```

Comment:
`log.Fatal` invokes `os.Exit(1)`, so I think the program will exit at this point and not flow through to `fmt.Fprintln(rw, result)`?